src/extensions/replay/sessionrecording.ts

```diff
@@ -337,14 +337,20 @@ export class SessionRecording {
         const canvasRecording_client_side = this.instance.config.session_recording.captureCanvas
         const canvasRecording_server_side = this.instance.get_property(SESSION_RECORDING_CANVAS_RECORDING)
 
-        const enabled = canvasRecording_client_side?.recordCanvas ?? canvasRecording_server_side?.enabled ?? false
-        const fps = canvasRecording_client_side?.canvasFps ?? canvasRecording_server_side?.fps ?? 0
-        const quality = canvasRecording_client_side?.canvasQuality ?? canvasRecording_server_side?.quality ?? 0
+        const enabled: boolean =
+            canvasRecording_client_side?.recordCanvas ?? canvasRecording_server_side?.enabled ?? false
+        const fps: number = canvasRecording_client_side?.canvasFps ?? canvasRecording_server_side?.fps ?? 4
+        let quality: string | number =
+            canvasRecording_client_side?.canvasQuality ?? canvasRecording_server_side?.quality ?? 0.4
+        if (typeof quality === 'string') {
+            const parsed = parseFloat(quality)
+            quality = isNaN(parsed) ? 0.4 : parsed
+        }
 
         return {
             enabled,
-            fps: clampToRange(fps, 0, 12, 'canvas recording fps'),
-            quality: clampToRange(quality, 0, 1, 'canvas recording quality'),
+            fps: clampToRange(fps, 0, 12, 'canvas recording fps', 4),
+            quality: clampToRange(quality, 0, 1, 'canvas recording quality', 0.4),
```

Comment:
Could these be constants? I have no idea what a quality of `0.4` means 😓

Reply:
done! 👍